packages/core/src/Stores/gtm-store.js

```diff
@@ -96,14 +96,127 @@ export default class GTMStore extends BaseStore {
+                    balance: response.transaction.balance,
+                },
+            };
+            Object.assign(data, extra_data);
+            this.pushDataLayer(data);
+
+            transactions_arr.push(response.transaction.transaction_id);
+            gtm_transactions.transactions = transactions_arr;
+            gtm_transactions.timestamp    = gtm_transactions.timestamp || moment_now.unix();
+
+            localStorage.setItem(storage_key, JSON.stringify(gtm_transactions));
+        }
+    }
+
+    @action.bound
+    eventHandler(get_settings) {
+        if (!this.is_gtm_applicable) return;
+
+        const login_event       = localStorage.getItem('GTM_login');
+        const is_new_account    = localStorage.getItem('GTM_new_account') === '1';
+
+        localStorage.removeItem('GTM_login');
+        localStorage.removeItem('GTM_new_account');
+
+        const affiliate_token = Cookies.getJSON('affiliate_tracking');
+        if (affiliate_token) {
+            this.pushDataLayer({ affiliate_token: affiliate_token.t });
+        }
+
+        // Get current time (moment, set by server), else fallback to client time
+        const moment_now = toMoment();
+        const data = {
+            visitorId     : this.visitorId,
+            account_type  : this.root_store.client.account_type,
+            currency      : this.root_store.client.currency,
+            country       : get_settings.country,
+            country_abbrev: get_settings.country_code,
+            email         : get_settings.email,
+            url           : window.location.href,
+            today         : moment_now.unix(),
+        };
+
+        if (is_new_account) {
+            data.event = 'new_account';
+            data.bom_date_joined = data.bom_today;
+        }
+
+        if (!this.root_store.client.is_virtual) {
+            data.bom_age       = moment_now.diff(epochToMoment(get_settings.date_of_birth), 'year');
+            data.bom_firstname = get_settings.first_name;
+            data.bom_lastname  = get_settings.last_name;
+            data.bom_phone     = get_settings.phone;
+        }
+
+        if (login_event) {
+            data.event = login_event;
+            BinarySocket.wait('mt5_login_list').then((response) => {
+                (response.mt5_login_list || []).forEach((obj) => {
+                    const acc_type = (getMT5AccountType(obj.group) || '')
+                        .replace('real_vanuatu', 'financial').replace('vanuatu_', '').replace(/svg/, 'gaming'); // i.e. financial_cent, demo_cent, demo_gaming, real_gaming
+                    if (acc_type) {
+                        data[`mt5_${acc_type}_id`] = obj.login;
+                    }
+                });
+                this.pushDataLayer(data);
+            });
+        } else {
+            this.pushDataLayer(data);
+        }
+
+        // check if there are any transactions in the last 30 days for UX interview selection
```

Comment:
remove all related to UX interviews. not used anymore

Reply:
@ashkanx This is (apparently) the only one. Is there any other you are aware of?